guides/learning_guide.rst

```diff
@@ -0,0 +1,1094 @@
+portability required for urgent decision making. Flux's fully hierarchical resource
+management allows this calss of workflows to run many modeling application tasks,
+each with a different scenario, efficiently at once on a large resource allocation.
+With strict deadline requirements, these workflows must also portably leverage
+computing resources from multiple institutions include the world's most powerful
+supercomputers at ORNL, LLNL, and NERSC. Existing software tools (e.g., SLURM
+and IBM LSF fall short of meeting all of the project's requirements - high job
+throughput, co-scheduling ability, and portability between different HPC systems. Luc
+Peterson, software architect of a main tool used for the NVBL team reported, "With
+Flux, we can model on scenario with UQ [uncertainty quantification] for the entire
+country in about five minutes on a few [Lassen supercomputer] nodes - that is, near
+real-time feedback.” This allowed U.S. policy decision makers to be informed of the
+results of COVID-19 spread modeling including confidence levels with no delay.
+
+Fast ML-Based COVID-19 Antiviral Drug Design
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In May 2020, a multidisciplinary LLNL team set out to develop a new highly scalable,
+end-to-end antiviral drug design workflow that could expediently produce potential
+COVID-19 drug molecules for further clinical testing. This team brought together
+multiple scientific experts.
+
+• LBANN (Livermore Big Artificial Neural Network Toolkit) researchers focused on developing a scalable ML technique to construct a high-quality ML model using 1.6 billion chemical compounds.
+
+• ATOM (Accelerating Therapeutics for Opportunities in Medicine) researchers coupled this ML model with a generative molecular design (GMD) pipeline to increase the probability of creating new COVID-19 antiviral drug molecules with desired properties (e.g., diversity, validity, efficacy, safety).
+
+• ConveyorLC researchers focused on coupling the above with their HPC simulations that evaluate the docking properties of the newly generated drug candidate molecules—that is, searching for an appropriate ligand that both energetically and geometrically fits the target protein's binding site.
+
+• Workflow infrastructure researchers devised workflow management techniques to ensure the scalability of this newly envisioned coupled workflow.
+
+The team quickly discovered that creating an end-to-end solution based on
+existing components could present two workflow issues - and that Flux could
+comprehensively solve them. First, the ConveyorLC docking simulation implements
+its own ad hoc task scheduling on top of message passing interface (MPI)
+functionality, which is not scalable. The team discovered that the scalability of this
+scheme indeed led to a significant resource utilization loss starting at 50 nodes. At
+200 nodes, the highest scale evaluated this scheduling scheme, resource utilization
```

Comment:
I think this is a typo from the pdf, should be "evaluated with this scheduling scheme" ... do we correct what we believe are errors in it?

Reply:
Yes I think so!